docs/source/model-export/export-mnn-zipformer.rst

```diff
@@ -0,0 +1,188 @@
+
+
+.. code-block:: bash
+
+  cd egs/librispeech/ASR
+  git clone https://huggingface.co/meixu/k2fsa-zipformer-bilingual-zh-en-t
+
+  cd ..
+
+In the above code, we downloaded the pre-trained model into the directory
+``egs/librispeech/ASR/k2fsa-zipformer-bilingual-zh-en-t``.
+
+.. _export_for_mnn_install_mnn:
+
+2. Install MNN
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+  # We put MNN into $HOME/open-source/MNN
+  # You can change it to anywhere you like
+
+  cd $HOME
+  mkdir -p open-source
+  cd open-source
+
+  git clone https://github.com/alibaba/MNN
+  cd MNN
+  mkdir build && cd build
+
+  cmake \
+    -DMNN_BUILD_CONVERTER=ON \
+    -DMNN_BUILD_TORCH=ON \
+    -DMNN_BUILD_TOOLS=ON \
+    -DMNN_BUILD_BENCHMARK=ON \
+    -DMNN_EVALUATION=ON \
+    -DMNN_BUILD_DEMO=ON \
+    -DMNN_BUILD_TEST=ON \
+    -DMNN_BUILD_QUANTOOLS=ON
+  ..
+
+  make -j4
+
+  cd ..
+
+  # Note: $PWD here is $HOME/open-source/MNN
+
+  export PATH=$PWD/build:$PATH
+
+Congratulations! You have successfully installed the following components:
+
+  - ``MNNConvert``, which is an executable located in
+    ``$HOME/open-source/MNN/build``. We will use
+    it to convert models from ``ONNX``.
+
+
+3. Export the model to ONNX
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+First, let us rename our pre-trained model:
+
+.. code-block::
+
+  cd egs/librispeech/ASR
+
+  cd k2fsa-zipformer-bilingual-zh-en-t/exp
+
+  ln -s pretrained.pt epoch-99.pt
+
+  cd ../..
+
+Next, we use the following code to export our model:
+
+.. code-block:: bash
+
+  dir=./k2fsa-zipformer-bilingual-zh-en-t
+
+  ./pruned_transducer_stateless7_streaming/export-onnx-zh.py \
+    --tokens $dir/data/lang_char_bpe/tokens.txt \
+    --exp-dir $dir/exp \
+    --use-averaged-model 0 \
+    --epoch 99 \
+    --avg 1 \
+    --decode-chunk-len 32 \
+    --num-encoder-layers "2,2,2,2,2" \
+    --feedforward-dims "768,768,768,768,768" \
+    --nhead "4,4,4,4,4" \
+    --encoder-dims "256,256,256,256,256" \
+    --attention-dims "192,192,192,192,192" \
+    --encoder-unmasked-dims "192,192,192,192,192" \
+    --zipformer-downsampling-factors "1,2,4,8,2" \
+    --cnn-module-kernels "31,31,31,31,31" \
+    --decoder-dim 512 \
+    --joiner-dim 512
+
+.. caution::
+
+  If your model has different configuration parameters, please change them accordingly.
+
+.. hint::
+
+  We have renamed our model to ``epoch-99.pt`` so that we can use ``--epoch 99``.
+  There is only one pre-trained model, so we use ``--avg 1 --use-averaged-model 0``.
+
+  If you have trained a model by yourself and if you have all checkpoints
+  available, please first use ``decode.py`` to tune ``--epoch --avg``
+  and select the best combination with with ``--use-averaged-model 1``.
+
+After the above step, we will get the following files:
+
+.. code-block:: bash
+
+  ls -lh k2fsa-zipformer-bilingual-zh-en-t/exp/*.onnx
+
+  .rw-rw-r--  88,435,414 meixu 2023-05-12 10:05 encoder-epoch-99-avg-1.onnx
```

Comment:
The displayed unit would be of unit `MB` if you use `-lh`. Please repost the result according to the command you are using.